inngest/src/main/kotlin/com/inngest/ktor/Route.kt

```diff
@@ -62,21 +62,15 @@ fun Route.serve(
                 val body = call.receiveText()
                 try {
                     val response = comm.callFunction(fnId, body)
-                    call.response.header(
-                        HttpHeaders.ContentType,
-                        ContentType.Application.Json.toString(),
-                    )
+                    response.headers.forEach({ (k, v) -> call.response.header(k, v) })
```

Comment:
does this loop also set content type to `application/json`?

Reply:
right the returned headers should always include a `content-type` header:
https://github.com/inngest/inngest-kt/blob/0c1f5981d1cd48800c9063f00ad7b1393c158349/inngest/src/main/kotlin/com/inngest/Environment.kt#L3-L11